daemon/regmaild/regmaild.c

```diff
@@ -607,6 +849,47 @@ client_cb(int fd, short event, void *arg)
             break;
         }
 
+        case VERIFYDB_REQ_COUNT:
+        {
+            const verifydb_req *req = &storage.verifydb;
+            if (!validate_verifydb_request(req))
+                goto end;
+
+            verifydb_count_rep rep = {};
+            verifydb_count(req, &rep);
+            fprintf(stderr, "verifydb count: %s generation [%ld] vmethod [%d] vkey [%s] "
+                    "(status: %d, self: %d, other %d)\n",
+                    req->userid, req->generation, req->vmethod, req->vkey,
+                    rep.status, rep.count_self, rep.count_other);
+            if (towrite(fd, &rep, sizeof(rep)) != sizeof(rep))
+                fprintf(stderr, " error: cannot write response...\n");
+            break;
+        }
+
+        case VERIFYDB_REQ_SET:
+        {
+            const verifydb_req *req = &storage.verifydb;
+            if (!validate_verifydb_request(req))
+                goto end;
+
+            int ret = verifydb_set(req);
+            fprintf(stderr, "verifydb set: %s generation [%ld] vmethod [%d] vkey [%s] (status: %d)\n",
+                    req->userid, req->generation, req->vmethod, req->vkey, ret);
+            if (towrite(fd, &ret, sizeof(ret)) != sizeof(ret))
+                fprintf(stderr, " error: cannot write response...\n");
+            break;
+        }
+
+        case VERIFYDB_REQ_GET:
+        {
+            const verifydb_req *req = &storage.verifydb;
+            if (!validate_verifydb_request(req))
+                goto end;
+
+            verifydb_get(req, fd);
+            goto end_noclose;
```

Comment:
Why the difference? Please keep all handling consistent.

Reply:
Fixed.